src/transport/AdminPairingTable.h

```diff
@@ -139,6 +146,20 @@ class DLL_EXPORT AdminPairingInfo
     CHIP_ERROR SetICACert(const chip::ByteSpan & cert);
     CHIP_ERROR SetRootCert(const chip::ByteSpan & cert);
 
+    const uint8_t * GetDeviceDAC(uint16_t & size)
+    {
+        size = mDeviceDACLen;
+        return mDeviceDAC;
+    }
+    const uint8_t * GetDevicePAI(uint16_t & size)
+    {
+        size = mDevicePAILen;
+        return mDevicePAI;
+    }
+
+    CHIP_ERROR SetDeviceDAC(const chip::ByteSpan & dac);
+    CHIP_ERROR SetDevicePAI(const chip::ByteSpan & pai);
```

Comment:
As commented by @pan-apple, this should be in Device, not AdminPairingTable.

Reply:
done